pkgs/applications/networking/cluster/pinniped/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec{
+  pname = "pinniped";
+  version = "0.17.0";
+
+  src = fetchFromGitHub {
+    owner = "vmware-tanzu";
+    repo = "pinniped";
+    rev = "v${version}";
+    sha256 = "sha256-0h7zyKe2gmC1n9EB5FRVI/io7Yj+91ZAtLy+1u3gyO0=";
+  };
+
+  subPackages = "cmd/pinniped";
+
+  vendorSha256 = "sha256-8ohyyciL1ORYOxPu64W0jXASTv+vVZR8StutzbF9N4Y=";
+
+  meta = with lib; {
+    description = "Pinniped is the easy, secure way to log in to your Kubernetes clusters.";
```

```suggestion
    description = "Tool to securely log in to your Kubernetes clusters";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

---

thanks, done